src/stylesheets/core/mixins/_alert-status-styles.scss

```diff
@@ -0,0 +1,29 @@
+@mixin alert-status-styles($name, $bgcolor) {
+  @include add-background-svg("alerts/#{$name}");
+  background-color: color($bgcolor);
+  $banner-text-color-token: get-token-from-bg($bgcolor);
+  color: color($banner-text-color-token);
+
+  &::before {
+    background-color: color(map-get($alerts-bar, $name));
+  }
+
+  .usa-alert__body {
+    padding-left: units($theme-alert-icon-size) + units($theme-alert-padding-x);
+  }
+
+  .usa-alert__text,
+  .usa-list {
+    a {
```

Comment:
Would `.usa-link` be more applicable here? Or even just `a` selector? If so, would need to check for specificity issues.

Reply:
That's a great find! I've updated to use `.usa-link`!